CampusTour/Data/ParseTag.swift

```diff
@@ -20,6 +21,8 @@ extension EventTag {
     }
 }
 
+//**** MARK ****
```

Comment:
to use `MARK` it has to be in the format of `//MARK: title` or `//MARK: - title` for the IDE to recognize it